ci-operator/step-registry/ipi/install/vsphere/virt/ipi-install-vsphere-virt-commands.sh

```
@@ -131,8 +127,10 @@ oc create -f "${SHARED_DIR}/vm.yaml" --kubeconfig=${VIRT_KUBECONFIG}
 
 # Update VM to have CI network
 echo "$(date -u --rfc-3339=seconds) - Patching networking config into virtual machines"
+export CLUSTER_NETWORK="$(oc get infrastructure cluster -o json --kubeconfig=${CLUSTER_KUBECONFIG} | jq -r '.spec.platformSpec.vsphere.failureDomains[0].topology.networks[0]')"
+echo "$CLUSTER_NETWORK"
 for (( i=0; i<${BM_COUNT}; i++ )); do
```

Shellcheck (0.11.0)

[style] $/${} is unnecessary on arithmetic variables.

(SC2004)